_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
+- Current portals and accounts
+- OneGC
+- API Store
+- Pan-Canadian Trust Framework
+- Some benefits integrated with 3rd parties
+
 ## Businesses
 
+- Able to find and integrate with 50% of ESDC services
+
```

Comment:
More customer-centric approaches may be used by private sector and emphasize may towards using IoT and data driven services. Could be spot some of the ESDC services that may be augmented by it? eg: Employment insurance, etc.

Reply:
Also, businesses may be experimenting with new technologies? Are there any technologies that may be near to making an impact in the next 5 years? (adapted from Jayson's thought this morning)

Reply:
I certainly hope so! 

Though we should stay focused on IT. So any new technologies IITB could use to better deliver services (ex IoT devices) should definitely be considered!)